libvirt/tests/src/daemon/conf_file/qemu_conf/set_virtlogd.py

```
@@ -1087,8 +1111,15 @@ def check_reload_max_backups():
             if with_console_log:
                 check_info_in_vm_log_file(vm_name, guest_log_file,
                                           matchedMsg=matched_msg)
+            elif expected_result == "destroy_vm_remove_logs":
+                if stdio_handler != "file" and os.path.exists(guest_log_file):
+                    test.fail('Guest qemu log should not exist after destroy with --remove-logs.')
```

There are only two values `file` and `logd`. This test case / behavior is only available with `logd`. As such, I prefer you use `stdio_handler == logd`. You might want to raise an exception if the stdio_handler is neither value but I think it's not necessary.